{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -1,44 +1,43 @@
 Django==1.11.18
-pytz==2018.7
-pylibmc==1.5.2
+pytz==2018.9
+pylibmc==1.6.0
```

Comment:
When I resolve the conflict here resulting in:
```
Django==1.11.20
pytz==2018.9
pylibmc==1.6.0
```

The build breaks, I think I've narrowed it down to `pylibmc==1.6.0` (by changing one thing at a time).
`src/_pylibmcmodule.h:42:10: fatal error: 'libmemcached/memcached.h' file not found`
I guess its because `pylibmc 1.6.0` is not totally compatible with `memcached 1.5`? (https://pypi.org/project/pylibmc/1.6.0/)
Just thought I'd flag/check its just me that needs to upgrade `memcached`.

Reply:
The version is fine, it's just the compilation of it which is the annoying issue.

pylibmc with Macports libmemcached needs the `LIBMEMCACHED` environment variable, however tox by default strips nearly all environment variables. If you work on a project which installs pylibmc outside of tox, it'll cache the compiled output and not cause a problem.

However to be correct here, I've tweaked the tox files a bit.